### ISSUE TITLE
Im so sorry for the trouble I just realized I was using numpy 1.16 for whatever reason. I just downloaded 1.15.4 and it works fine now. Thanks!

### ISSUE THREAD

**Issue by EliyaShtraser** [scikit-image/scikit-image#4303]
Im so sorry for the trouble I just realized I was using numpy 1.16 for whatever reason. I just downloaded 1.15.4 and it works fine now. Thanks!

_Originally posted by @tw255 in https://github.com/scikit-image/scikit-image/issues/3586#issuecomment-451345803_

I had numpy version = 1.17. i did it and it work now

**Comment by hmaarrfk** [comment]
??? did you have an issue with numpy 1.17? That shouldn't happen. I think we support numpy 1.17 with the latest version fo scikit-image

**Comment by EliyaShtraser** [comment]
Hi hmaarrfk,
I'm using colab. when I run the following lines I see this error message.
```
import numpy as np
print(np.__version__)
from numpy.lib.arraypad import _validate_lengths
```
The error message is.

> 1.17.4
> 
> ---------------------------------------------------------------------------
> 
> ImportError                               Traceback (most recent call last)
> 
> <ipython-input-4-0536638a2771> in <module>()
>       1 import numpy as np
>       2 print(np.__version__)
> ----> 3 from numpy.lib.arraypad import _validate_lengths
> 
> ImportError: cannot import name '_validate_lengths'
> 
> 
> ---------------------------------------------------------------------------
> NOTE: If your import is failing due to a missing package, you can
> manually install dependencies using either !pip or !apt.
> 
> To view examples of installing some common dependencies, click the
> "Open Examples" button below.

So I downgrade the numpy version to 1.15.4 by this command.
! pip install numpy==1.15.4
 And everything is OK.

**Comment by jni** [comment]
Ah, that issue could also be fixed by upgrading scikit-image to 0.14.5, 0.15.0, or 0.16.2.
